src/extra_files.c

```diff
@@ -148,9 +149,19 @@ int walk_tree(struct manifest *manifest, const char *start)
 			if (F[i].dir) { /* Start of new dir to skip */
 				skip_dir = F[i].filename;
 				skip_len = strlen(skip_dir);
-				printf("%s/\n", F[i].filename);
+				if (fix) {
+					printf("REMOVING DIR %s/\n", F[i].filename);
+					remove(F[i].filename);
+				} else {
+					printf("%s/\n", F[i].filename);
+				}
 			} else {
-				printf("%s\n", F[i].filename);
+				if (fix) {
+					printf("REMOVING %s\n", F[i].filename);
+					remove(F[i].filename);
```

Comment:
Thanks for pointing this out, this has caused problems in mixer before when not properly handled.